src/rai_bench/rai_bench/tool_calling_agent/results_tracking.py

```
@@ -41,8 +41,12 @@ class ValidatorResult(BaseModel):
 
 
 class TaskResult(BaseModel):
-    task_prompt: str = Field(..., description="The task prompt.")
+    task_prompt: str = Field(..., description="The task base prompt.")
     system_prompt: str = Field(..., description="The system prompt.")
+    examples_in_system_prompt: int = Field(
+        ..., description="Number of examples on how to use tool in system prompt"
```

Flake8 (7.2.0)

[error] line too long (81 > 79 characters)

(E501)